includes/i18n.php

```diff
@@ -186,6 +186,7 @@
 		'placeholder-what-did-you-expect'          => __( "What did you expect?", 'freemius' ),
 		'reason-didnt-work'                        => __( "The plugin didn't work", 'freemius' ),
 		'reason-dont-like-to-share-my-information' => __( "I don't like to share my information with you", 'freemius' ),
+		'dont-have-to-share-any-data'              => __( "You might have missed it, but you don't have to share any data and can just %s the opt-in. %s", 'freemius' ),

```

Comment:
@fajardoleo the suffix of ` %s` is no needed. There's no reason to enforce the next button to be part of the sentence. Just append the string where you use it.